cogs/cog_inspire.py

```diff
@@ -3,19 +3,34 @@
 from models import inspire
 
 
+# This cog utilizes the Inspire object to send appropriate quote based on user input
 class InspireCommands(commands.Cog, name="Inspire Commands"):
 
     def __init__(self, bot):
         self.bot = bot
 
+    # This command returns satirical quotes related to computer programming
     @commands.command(name="get-cs-satire", aliases=["css"])
-    async def display_quote(self, ctx):
-
-    #create a quote oject, call the appropriate method and split by author/content
+    async def display_quote(self, ctx, discord_role: str = ''):
+        # sanitize input and check if parameter is empty string or contains role
+        check_param: bool
+        if '<@&' in discord_role and '>' in discord_role:
+            check_param = True
+        elif discord_role == '':
+            check_param = False
+        else:
```

Comment:
We can simplify this to just check if `discord_role` is empty or not using a single boolean expression.